src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let cashBack = 0;
+
+  if (days > 0 && days < 3) {
+    return rent * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    cashBack = 20;
+
+    return rent * days - cashBack;
+  }
+
+  if (days >= 7) {
+    cashBack = 50;
+
+    return rent * days - cashBack;
+  }
```

Comment:
You don't need this if statement, because previous statements cover every other scenario.
If you divide cashback into discounts - you can just return it like that

```suggestion
    return rent * days - maxDiscount;
```